connector.py

```diff
@@ -0,0 +1,59 @@
+"""
+Copyright 2019 Google LLC
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+  https://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+"""
+
+
+def get_ephemeral(service, project_name, instance_name, pub_key):
+    """
+    A helper function that requests an ephemeral certificate from the
+    Cloud SQL Instance.
+
+    Args:
+        service (googleapiclient.discovery.Resource): A service object created
+          from the Google Python API client library. Must be using the SQL
```

Comment:
Perhaps we should mention the package, or have a link to the documentation? Often users are confused between the gRPC and the Rest/Discovery client libraries.

Reply:
@fatcat2 Would you mind linking to the repo: https://github.com/googleapis/google-api-python-client ?

I'm currently in the process of moving around documentation for that library. :)